installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+[id="preparing-to-install-on-vsphere"]
+= Preparing to install on vSphere
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-vsphere
+
+toc::[]
+
+
+[id="preparing-to-install-on-vsphere-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+* You provisioned xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] for your cluster. To deploy a private image registry, your storage must provide
+`ReadWriteMany` access modes.
+
+* If you use a firewall and plan to use Telemetry, you
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured the firewall to allow the sites] required by your cluster.
+
+* For user-provisioned infrastructure, xref:../../installing/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[you created a registry on your mirror host] and obtained the `imageContentSources` data for your version of {product-title}.
```

Comment:
This prerequisite is only relevant to the installation assemblies that use restricted networks. It is not a requirement in all OCP on vSphere deployment scenarios.

Reply:
Removed. Thank you!